src/pug/blog/report_25.pug

```diff
@@ -0,0 +1,98 @@
+extends _blog.pug
+
+block append subtitle
+  | Progress report #25
+
+block content
+  h2 Progress report #25
+  +news_header('13 September 2024', 'report_24', '')
+
+  h3 Wireless RF prototypes 6 and 7
+  p We continued optimizing antenna performance by doing #[em "trim tuning"], adding a long copper stub on the antenna's end, and then taking measurements while progressive scratching off the copper and reducing the stub length.
```

Comment:
Maybe not "optimizing", as that may imply that we have a viable solution, but instead "experimenting".
Rephrase, structure into multiple sentences.